From 3aab4694f97f96fc7d3608fb3a7bff047e390b4e Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Tue, 14 Mar 2023 16:07:28 +0100
Subject: [PATCH] Revert "test: accept EPERM for unavailable idmapped mounts as
 well"

This reverts commit d2ccb0bf2b2078e13fc4cb1c2e2eb8079fa57df0.

Related: #2178222
---
 test/units/testsuite-13.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/units/testsuite-13.sh b/test/units/testsuite-13.sh
index 2b6e6df8a3..4ad7431e42 100755
--- a/test/units/testsuite-13.sh
+++ b/test/units/testsuite-13.sh
@@ -74,7 +74,7 @@ function check_rootidmap {
             --register=no -D "$_root" \
             --bind=/tmp/rootidmapdir:/mnt:rootidmap \
             /bin/sh -c "$_command" |& tee nspawn.out; then
-        if grep -Eq "Failed to map ids for bind mount.*: (Function not implemented|Operation not permitted)" nspawn.out; then
+        if grep -q "Failed to map ids for bind mount.*: Function not implemented" nspawn.out; then
             echo "idmapped mounts are not supported, skipping the test..."
             return 0
         fi
